src/extensions/default/BrambleUrlCodeHints/main.js

```diff
@@ -213,7 +248,13 @@ define(function (require, exports, module) {
         unfiltered.forEach(function (item) {
             if(isImage) {
                 if(Content.isImage(Path.extname(item))) {
+                    if(Array.isArray(item)){
```

Comment:
You don't need to bother checking if this is an array, you an just use `Array.prototype.concat` and it will handle both cases:

```
> var result = []
> result
[]
> result = result.concat("simple string")
[ 'simple string' ]
> result = result.concat(["array", "of", "strings"])
[ 'simple string', 'array', 'of', 'strings' ]
```

You should be able to just do `result = result.concat(item);` and get rid of all this.